tests/lib/rules/no-loop-func.js

```diff
@@ -111,7 +111,65 @@ ruleTester.run("no-loop-func", rule, {
                 "let a;"
             ].join("\n"),
             parserOptions: { ecmaVersion: 6 }
+        },
+
+        /*
+         * These loops _look_ like they might be unsafe, but because i is undeclared, they're fine
+         * at least as far as this rule is concerned - the loop doesn't declare/generate the variable.
+         */
+        {
+            code: "while(i) { (function() { i; }) }",
+            errors: [{ messageId: "unsafeRefs", data: { varNames: "'i'" }, type: "FunctionExpression" }]
```

Comment:
```suggestion
            code: "while(i) { (function() { i; }) }"
```

There should be no `errors` property in `valid` tests.

Reply:
Yup, silly me cut+pasting from the `invalid` section and forgetting to chop off the errors 😄